src/server/pfs/cmds/cmds.go

```diff
@@ -265,6 +259,59 @@ $ pachctl start-commit test -p XXX
 	startCommit.Flags().StringVarP(&description, "message", "m", "", "A description of this commit's contents")
 	startCommit.Flags().StringVar(&description, "description", "", "A description of this commit's contents (synonym for --message)")
 
+	startCommits := &cobra.Command{
+		Use:   "start-commits (<repo-name> <branch-name> [flags]) ...",
+		Short: "Start multiple new commits on multiple repos.",
+		Long:  "Start multiple new commits on multiple repos.  Each commit will be started on the specified branch; if the branch does not exist, it will be created.  A parent commit may be specified to fork from an existing commit.",
+		DisableFlagParsing: true,
+	}
+	startCommits.Flags().StringP("parent", "p", "", "The parent of the new commit, unneeded if branch is specified and you want to use the previous head of the branch as the parent.")
+	startCommits.Flags().StringP("message", "m", "", "A description of this commit's contents")
+	startCommits.Flags().String("description", "", "A description of this commit's contents (synonym for --message)")
+
+	startCommits.Run = cmdutil.RunBatchCommand(
+		2, // New batch for every two positional args
+		func (argSets []cmdutil.BatchArgs) error {
+			requests := []*pfsclient.StartCommitRequest{}
+
+			for _, args := range argSets {
+				// TODO: if multiple of these are specified, we won't know which one was the latest
+				// There are no aliases for flags, but we may be able to override annotations to link the two
```

Comment:
This is a known issue (as a side-effect of having two flags for the same value) - I could fix it here, but I don't think it's a very big issue, just thought I'd raise awareness.